openssl/src/ssl/mod.rs

```diff
@@ -282,6 +320,12 @@ impl SslContext {
         }
     }
 
+    pub fn set_read_ahead(&self, m: c_long) {
+        unsafe {
+            ffi::SSL_CTX_set_read_ahead(*self.ctx, m);
```

Comment:
You're right, I changed it to u32. usize semantically implies that it would be a size.